deployment/helm/node-feature-discovery/values.yaml

```
@@ -529,7 +529,10 @@ gc:
 # need to manually, or otherwise, provision the TLS certs as secrets
 tls:
   enable: false
-  certManager: false
+  certManager:
+    enable: false
+    issuerKind:
+    issuerName:
```

We shouldn't break existing users


```suggestion
  certManager: false
  certManageIssuerKind:
  certManageIssuerName:
```

OR

```suggestion
  certManager: false
  certManagerCertificate:
    issuerKind:
    issuerName:
```